compiler-rt/lib/sanitizer_common/sanitizer_file.h

```diff
@@ -79,7 +79,8 @@ bool SupportsColoredOutput(fd_t fd);
 const char *GetPwd();
 bool FileExists(const char *filename);
 bool DirExists(const char *path);
-char *FindPathToBinary(const char *name);
+char *FindPathToBinaryOrLibrary(const char *name,
+                                const char *env_string = "PATH");
```

Comment:
```suggestion
char *FindPathToBinaryOrLibrary(const char *name, const char *env_string);
inline char *FindPathToBinary(const char *name) {
  return FindPathToBinaryOrLibrary(name, "PATH");
}
```

Reply:
See additional downstream discussion on the procfs-based implementation for why we might not need `FindPathToBinaryOrLibrary` at all.